cc_licenses/templates/base.html

```diff
@@ -22,9 +22,9 @@
           {% include 'includes/skiplink.html' %}
           <nav class="breadcrumb level-left caption bold" aria-label="breadcrumbs">
             <ul>
-                <li><a href="#">{% trans "Home" %}</a></li>
-                <li><a href="#">{% trans "License" %}</a></li>
-                <li class="is-active"><a href="#" aria-current="page displayed">{% block page-displayed %}{% endblock %}</a></li>
+                <li><a href="https://creativecommons.org/">{% trans "Home" %}</a></li>
+                <li><a href="https://creativecommons.org/licenses/">{% trans "Licenses" %}</a></li>
```

Comment:
Maybe these links should go to ```/``` and ```/licenses/```, so if we're testing on staging or a preview site for example, we're not linking off to a different site?

Reply:
sounds good

Reply:
done